### PR TITLE
Change Karavi references to CSM.

### PR DESCRIPTION
**Pull request by ameer-jabbar**
# Description
Update the documentation to reflect the new naming of the product under the Container Storage Modules (CSM) umbrella. 
There were also few typos and grammatical updates included with the changes.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| #46 |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

# How Has This Been Tested?
Documentation updates only.

### CODE REVIEW

**Review by rbo54 — APPROVED** [review]
You found and corrected a lot of typos we didn't catch in the last review. Thanks!

**Review by tdawe — DISMISSED** [review]

**Comment by ameer-jabbar** [comment]
> You found and corrected a lot of typos we didn't catch in the last review. Thanks!

The IDE helped a lot with that :)

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
 -->
 
-# Karavi Resiliency
+# Dell EMC Container Storage Modules (CSM) for Resiliency
```
Comment (gallacher) [inline README.md:11]: Module instead of Modules

Comment (ameer-jabbar) [reply README.md]: Sure. Updated.

**Review by tdawe — APPROVED** [review]

**Review by gallacher — APPROVED** [review]

**Pull request merged**